index.html

```diff
@@ -0,0 +1,216 @@
+              A fun and modern office where people from different countries who believe in the value of development and
+              possitivity comes together to build and develop their thoughts.
+            </div>
+            <div class="hero-time h1 fw-bold text-sm-start text-center">
+              Sun - Mon 8am - 5pm (GMT+4:30)
+              <div class="clearfix"></div>
+              <span class="fw-lighter">@ Shahr-e-Naw Ansari Square, Kabul, Afghanistan</span>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="facilities">
+        <div class="container py-5 text-center">
+          <h2 class="h2 text-white mt-4">Our Facilities</h2>
+          <div class="small-border mx-auto"></div>
+          <div class="row justify-content-between gap-1 mt-5">
+            <div class="col-md col-12 card facility-card text-center py-4 px-3 rounded-0">
+              <div class="row align-items-center">
+                <span class="col-2 col-md-12 bi bi-wifi text-white h1"></span>
+                <h5 class="col-4 col-md-12 text-orange my-3">Fast Internet</h5>
+                <p class="col-6 col-md-12 text-white fs-7 my-auto">
+                  Use fast and dedicated internet on your working space.
+                </p>
+              </div>
+            </div>
+            <div class="col-md col-12 card facility-card text-center py-4 px-3 rounded-0">
+              <div class="row align-items-center">
+                <span class="col-2 col-md-12 bi bi-cup-straw text-white h1"></span>
+                <h5 class="col-4 col-md-12 text-orange my-3">Refreshment</h5>
+                <p class="col-6 col-md-12 text-white fs-7 my-auto">
+                  Get yourself refreshed with our basic refreshments: tea, coffee, water.
+                </p>
+              </div>
+            </div>
+            <div class="col-md col-12 card facility-card text-center py-4 px-3 rounded-0">
+              <div class="row align-items-center">
+                <span class="col-2 col-md-12 bi bi-playstation text-white h1"></span>
+                <h5 class="col-4 col-md-12 text-orange my-3">Games</h5>
+                <p class="col-6 col-md-12 text-white fs-7 my-auto">
+                  Enjoy from video games to with other co working space friends.
+                </p>
+              </div>
+            </div>
+            <div class="col-md col-12 card facility-card text-center py-4 px-3 rounded-0">
+              <div class="row align-items-center">
+                <span class="col-2 col-md-12 bi bi-people text-white h1"></span>
+                <h5 class="col-4 col-md-12 text-orange my-3">Meeting Room</h5>
+                <p class="col-6 col-md-12 text-white fs-7 my-auto">
+                  Fully furnished and up to 10 people accommodable space for your training sessions, events, and
+                  meetings.
+                </p>
+              </div>
+            </div>
```

Comment:
- [ ] Please make sure to Implement the hovering technique is in Behance design 

| Your design | Behance |
| ------------- | --------- |
| ![image](https://user-images.githubusercontent.com/74744787/163285834-138d8a1b-c904-4c23-ab99-3f08793b4046.png) | ![image](https://user-images.githubusercontent.com/74744787/163285872-7466680f-1a65-4b85-a1a6-44af55a3d227.png) |
if you notice the design you can see how the border turns to be white when the user do an over mouse event